CASMISNT-4800: add notes about how long it can take for NCN clocks to sync (#1840)
Cray-HPE/docs-csm: operations/utility_storage/Troubleshoot_System_Clock_Skew.md

Resynchronize system clocks after Ceph reports a clock skew.

Systems use `chronyd` to synchronize their system clocks. If systems are not able to communicate, then the clocks can drift,

> causing clock skew. Another reason for this issue would be an individual manually changing the clocks or a task that may change

Reviewer: ```suggestion
causing clock skew. Another reason for this issue could be an individual manually changing the clocks or a task that may change
```

> causing clock skew. Another reason for this issue would be an individual manually changing the clocks or a task that may change

Reviewer: ```suggestion
causing clock skew. Clock skew can also be caused by an individual or an automated task manually changing the clocks.
```

> the clocks and require a series of steps \(time adjustments\) to resynchronize.

Reviewer: ```suggestion
In this case, `chronyd` may require a series of steps \(time adjustments\) to resynchronize the clocks.
```
